CONTRIBUTING.md

```diff
@@ -1,27 +1,21 @@
 If you would like to contribute, please submit a PR. If you encounter an
 problem, please file an Issue in this repo.
 
-### Making updates
+### Releasing
 
-Actions need the compiled code checked in. This means your changes will only
-take effect if you run the build to generate .js files from the .ts Typescript
-files and check the .js files in as well.
-
-### Adding new dependencies
-
-Check in the Action with everything it needs to run. If you update dependencies
-you will need to commit the changes to Node modules as well.
+Releases are done of of release branches. Each major release version should get
```

Comment:
`of of` should be `on`? I would suggest rewording this to avoid using the word `done` because it can ambiguous as to what that means.